inc/functions/files.php

```diff
@@ -838,17 +841,18 @@ function rocket_clean_domain( $lang = '' ) {
 	 */
 	$urls = (array) apply_filters( 'rocket_clean_domain_urls', $urls, $lang );
 	$urls = array_filter( $urls );
```

Comment:
I think this is a small and maybe not effective change but wanted to highlight it.
I think looping on items to assure their values are better/faster than array_filter, I tried sample code to check and found that.
You can take this as reference: http://www.levijackson.net/are-array_-functions-faster-than-loops/

Reply:
@engahmeds3ed I remember this article. I had issues with how the benchmarking was done, though I didn't validate his findings through proper benchmarking.

What do I mean? Why did I take issues with it?

Benchmarking requires a large statistical sample of running the code over and over again and then taking the averages. Typically, we run these samples in 10k, 100k, and/or 1m cycles. 

Why? Each time we run it, the results vary. When dealing with fractions of a second, we need a large sample set to analyze and average.

But you raise a valuable point. We need to analyze and benchmark to optimize. Let's do that as part of our Filesystem module work. Sound good?

Reply:
Sounds great.